### PR TITLE
Updated Win install/uninstall to reflect handling of Path

### LINKED ISSUE #266: Appsody CLI on Windows - document new behavior of setup command
**Is your feature request related to a problem? Please describe.**
The setup command has changed the way it handles PATH and User Path env vars.

**Describe the solution you'd like**
Document the new behavior before the release goes out.

See this: https://github.com/appsody/appsody/pull/349

### PR DESCRIPTION
**Pull request by chilanti**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description

Describes the new behavior of the Windows installer with regard to handling the Path.

## Related Issues
Fixes #266

### CODE REVIEW

**Comment by chilanti** [comment]
@neeraj-laad - to be merged after we release.

File: content/docs/getting-started/installation.md [hunk 1]
```diff
@@ -179,4 +181,4 @@ sudo yum remove appsody
 You may also want to remove the `.appsody` directory that gets created under your home directory: `rm -rf ~/.appsody`.
 
 ## Uninstalling on Windows
-To uninstall Appsody, delete the directory where you extracted the binaries. Optionally, remove the `.appsody` directory that gets created under your home folder and the **PATH** entry that was created by the setup command (see [Modifying the PATH on Windows 10](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/)).
+To uninstall Appsody, delete the directory where you extracted the binaries. Optionally, remove the `.appsody` directory that gets created under your home folder and the **Path** User Variable entry that was created by the setup command (see [Modifying the PATH on Windows 10](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/)). You may also want to remove the **APPSODY_PATH** variable, from the same dialog used to update the Path variable.
```
Comment (neeraj-laad) [inline content/docs/getting-started/installation.md]: Not sure if that matches rest of the doc, but listing these steps as a list might make it clearer for users.

To uninstall Appsody: 
- delete the directory where you extracted the binaries
- remove the `.appsody` the directory under your home folder
- remove **APPSODY_PATH** from the **Path** User Variable entry (see [Modifying the PATH on Windows 10](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/)). You may also want to remove the **APPSODY_PATH** variable, from the same dialog used to update the Path variable.

File: content/docs/getting-started/installation.md [hunk 2]
```diff
@@ -77,7 +77,9 @@ appsody-setup.bat
...
 
-To uninstall Appsody, delete the directory where you extracted the binaries. Optionally, remove the `.appsody` directory that gets created under your home folder and the **PATH** entry that was created by the setup command (see [Modifying the PATH on Windows 10](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/)).
+Note that `appsody-setup.bat` modifies your **PATH** environment variable - adding the folder where Appsody is installed to it - and creates an **APPSODY_PATH** [User Environment Variable](https://docs.microsoft.com/en-us/windows/win32/shell/user-environment-variables), set to the folder where Appsody is installed. The **APPSODY_PATH** environment variable is also appended to the **Path** User Environment Variable. This makes it possible for your to run Appsody from the same session you used to install it, or from any command prompt session you may open in the future.
+
+If you run `appsody-setup.bat` multiple times, the script updates your **PATH** environment variable and the **APPSODY_PATH** variable, if necessary (i.e. if you ran it from a different directory).
```
Comment (neeraj-laad) [inline content/docs/getting-started/installation.md]: I think this section could be simplified a bit... How about something like this:

Note that `appsody-setup.bat` modifies the following [user environment variables](https://docs.microsoft.com/en-us/windows/win32/shell/user-environment-variables):
- creates an **APPSODY_PATH** variable, set to the folder where Appsody is installed
- Updates **PATH** environment variable, by adding **APPSODY_PATH**

Comment (neeraj-laad) [reply content/docs/getting-started/installation.md]: rest of the words around being able to run multiple times is an expectation anyways, I don't think we need to call it out explicitly.. Am I missing something?

File: content/docs/getting-started/installation.md (hunk 1 above)
Comment (chilanti) [reply content/docs/getting-started/installation.md]: Agree.

File: content/docs/getting-started/installation.md (hunk 2 above)
Comment (chilanti) [reply content/docs/getting-started/installation.md]: I'll make those changes - and yes - there's no point in mentioning running multiple times....

**Comment by neeraj-laad** [comment]
@appsody/content can this ber merged?

**Pull request merged**